app/controllers/subsidy/applications/edit.js

```diff
@@ -29,6 +33,107 @@ export default class SubsidyApplicationsEditController extends Controller {
     return this.model.consumption.get('status.isConcept');
   }
 
+  get canDownload() {
+    return !this.model.consumption.get('status.isConcept');
```

Comment:
Not 100% sure if this is the correct check. In my head it only makes sense to print a form that is in read-only mode. Not sure how hard that would be, and it's already better than always showing it at least.

For example, the button is displayed, but the form can still be edited:
![image](https://github.com/lblod/frontend-subsidiepunt/assets/3533236/e9fe3427-5e72-4fab-b72e-ac023c59ce83)

Opinions?

Reply:
I checked with Marie and we want to always show the button